src/panels/developer-tools/template/developer-tools-template.ts

```diff
@@ -140,8 +143,51 @@ class HaPanelDevTemplate extends LitElement {
             size="small"
           ></ha-circular-progress>
           <pre class="rendered ${classMap({ error: this._error })}">
```

Comment:
Make `_error` a string instead of an boolean, and render it when not undefined.
```suggestion
          <pre class="rendered ${classMap({ error: this._error })}">
```